NightscoutServiceKitUI/Views/ServiceStatusView.swift

```diff
@@ -39,6 +41,14 @@ struct ServiceStatusView: View, HorizontalSizeClassOverride {
                     Text(String(describing: viewModel.status))
                 }
                 .padding()
```

Comment:
I'm not a SwiftUI expert yet - I haven't figured out how to remove the blue text and get a chevron in place. Any suggestions welcome but not sure also if this is a blocker yet.

![Screen Shot 2021-09-03 at 5 08 13 PM](https://user-images.githubusercontent.com/3207996/132065471-db1eda05-9d1d-439c-9f0a-ddaacf0008ce.jpg)

Reply:
The issue seems to be that this button needs to be in a SwiftUI List  to get the default chevron button styling. I experimented with that in commit `9ac625a5c488e918dea8f573fe26302716c3873a`. That caused a different set of issues including the fact this List has a different background color than our our lists in Settings (inverted). I figured out how to change the row colors but not the background behind the list. Also the list would stay highlighted after popping back to this view, without adding a hack from Stack Overflow.

I opted for a simpler approach - With the latest update, I added a chevron image directly to the row and make sure the button text color is correct (non-blue). This is not ideal to not use the system scheme but seems pretty close to the desired layout.

<img width="334" alt="Screen Shot 2021-09-12 at 9 51 50 AM" src="https://user-images.githubusercontent.com/3207996/132990325-7e66234b-9b0c-4ce3-8b8f-e17f090b7ca2.png">